app/presenters/tree_builder_protect.rb

```diff
@@ -33,9 +33,16 @@ def x_get_tree_roots
   def x_get_tree_hash_kids(parent, count_only)
     nodes = parent[:nodes].map do |policy|
       icon = policy.towhat.safe_constantize.try(:decorate).try(:fonticon)
+      mode = if policy.mode == "control"
```

Comment:
I removed the `policy.description` change and added this if statement since I couldn't find anywhere where I could catalog the possible values of `policy.mode` with N_()